jenkins/plan.sh

```diff
@@ -12,7 +12,8 @@ pkg_exports=(
   [port]=jenkins.http.port
 )
 pkg_exposes=(port)
-pkg_svc_user="root"
+pkg_svc_user=hab
```

Comment:
Since these are the default they should be removed

Reply:
@predominant gentle ping

Reply:
Done @elliott-davis / @nellshamrell